### ISSUE TITLE
[Bug]: Updating backlight % is delayed

### ISSUE THREAD

**Issue by vaibd** [polybar/polybar#2570]
### Checklist

- [X] I have read the appropriate section in the [contributing guidelines](https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md)
- [X] I believe this issue is a problem with polybar itself and not a misconfiguration on my part
- [X] I have searched for other open and closed [issues](https://github.com/polybar/polybar/issues?q=is%3Aissue) that may have already reported this problem
- [X] I have checked the [known issues](https://github.com/polybar/polybar/wiki/Known-Issues) page for this problem.
- [X] I have followed the [debugging guide](https://github.com/polybar/polybar/wiki/Debugging-your-Config) to narrow down the problem to a minimal config.

### Steps to reproduce

When I update backlight using brightnessctl the value in polybar lags behind the actual value.
Also, when I inc the brightness and then dec it, it first increases and then decreases and vive-versa.

Lets say the brightness is 50% currently
Inc +5%, brightness value = 55%
Dec -5%, brightness value = 60% (here, it inc instead of decreasing)
Dec -5%, brightness value = 55%
It replicates the previous increment/decrement whenever new decrement/increment about to happen.

It seems like the problem is similar to: https://github.com/polybar/polybar/issues/2367
Also, `use-actual-brightness = true` or `use-actual-brightness = false` doesnt fix the issue.

### Minimal config

```dosini
[module/backlight]
type = internal/backlight
card = amdgpu_bl0
use-actual-brightness = true

output = eDP
enable-scroll = true
format =  <label>
format-foreground = ${colors.foreground}
format-background = ${colors.background}
label-foreground = ${colors.foreground}
label = %percentage%%
```

### Polybar log

```text
no error in logs
```

### Expected behavior

Brightness to update properly

### Actual behavior

Brightness to update properly

### Window Manager and Version

bspwm, 0.9.10

### Linux Distribution

Arch linux

### Polybar version

```text
polybar 3.5.7

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm +xcursor

Build type: Release
Compiler: /usr/bin/c++
Compiler flags: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Wextra -Wpedantic -O3 -DNDEBUG
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now
```

**Comment by patrick96** [comment]
That bug indeed looks similar to #2367 

However, this bug is not fixed in polybar 3.5.7, the fix is not part of any released version yet. Please try to use the `polybar-git` package and see if the issue persists.

**Comment by vaibd** [comment]
`polybar-git` works fine, thanks.
